config/initializers/leaflet.rb

```diff
@@ -1,3 +0,0 @@
-Leaflet.tile_layer = "http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-Leaflet.attribution = '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a>'
-Leaflet.max_zoom = 19
```

Comment:
(I add this comment here, to not crash anyone server when trying to display the lock file in the browser)

Why do we have a `package-lock.json` and a `yarn.lock`?
I thought we only use yarn. Therefore there shouldn't be a `package-lock.json` that is generated by npm?

Reply:
we didn't really touch that part. Package-lock has been around since the beginning I think